README.md

```diff
@@ -16,42 +64,72 @@ The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tr
 
 All of the released versions of the AWS IoT Embedded C SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
 
+## Versioning
+
+C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN_major, where: 
+
+* Y represents the year.
+* M represents the month.
+* N represents the release order within the designated month (00 being the first release).
+* A "major" denotation indicates the addition of new features or significant updates to multiple libraries. 
+
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application. 
+
+## Cloning
+This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to bring in dependent components.
```

Comment:
dependent?  Who depends upon whom?

Reply:
Fixed the statement to remove the "dependent" word